rust/processor/src/models/token_v2_models/v2_token_activities.rs

```diff
@@ -238,6 +239,28 @@ impl TokenActivityV2 {
                     transaction_timestamp: txn_timestamp,
                 }));
             } else {
+                // This should only happen in the case where we have a burn event and mint event in the same
+                // transaction.
+                if tokens_minted.get(&token_data_id).is_some() {
+                    return Ok(Some(Self {
+                        transaction_version: txn_version,
+                        event_index,
+                        event_account_address,
+                        token_data_id,
+                        property_version_v1: BigDecimal::zero(),
+                        type_: event_type,
+                        from_address: None,
+                        to_address: None,
+                        token_amount: BigDecimal::one(),
```

Comment:
shouldn't this be 0?

Reply:
just following what we do for burns on line 292. we set the token_amount to 1